tests/Resource/CloudAccount/BackupTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * @package Nexcess-SDK
+ * @license https://opensource.org/licenses/MIT
+ * @copyright 2018 Nexcess.net, LLC
+ */
+
+declare(strict_types  = 1);
+
+namespace Nexcess\Sdk\Tests\Resource\CloudAccount;
+
+use Nexcess\Sdk\ {
+  Resource\CloudAccount\Endpoint,
+  Resource\CloudAccount\Entity,
+  Resource\CloudAccount\Backup,
+  Resource\PromisedResource,
+  Resource\Collection,
+  Resource\Model,
+  Tests\Resource\ModelTestCase,
+  Util\Config
+};
+
+/**
+ * Unit test for cloud accounts (virtual hosting).
+ */
+class BackupTest extends ModelTestCase {
+
+  /** {@inheritDoc} */
+  protected const _RESOURCE_PATH = __DIR__ . '/resources';
+
+  /** {@inheritDoc} */
+  protected const _RESOURCE_FROMARRAY = 'cloud-account-1.fromArray.json';
+
+  /** {@inheritDoc} */
+  protected const _RESOURCE_TOARRAY = 'cloud-account-1.toArray-shallow.php';
+
+  /** {@inheritDoc} */
+  protected const _RESOURCE_TOCOLLAPSEDARRAY =
+    'cloud-account-1.toCollapsedArray.json';
+
+  /** {@inheritDoc} */
+  protected const _SUBJECT_FQCN = Backup::class;
+
+ 
+  /**
+   * @covers Backup::download
+   */
+  public function testDownloadBackup() {
+    $path = '/tmp';
+    $filename = 'filename.tgz';
+    $force = false;
+
+    $entity  = new Entity;
+    $entity->sync(['id'=>1]);
+
+    $backup = new Backup;
+    $backup->sync(['filename'=>$filename]);
+    $backup->setCloudAccount($entity);
+    
+    $endpoint = $this->createMock(Endpoint::class);
+    $endpoint->method('downloadBackup')
+      ->with(
+        $this->equalTo($entity),
+        $this->equalTo($filename),
+        $this->equalTo($path),
+        $this->equalTo($force)
+      );
+    $backup->setApiEndpoint($endpoint);
+    $backup->download($path);
+    $this->assertTrue(true);
```

Comment:
on its own, this doesn't prove `$endpoint->downloadBackup()` was invoked (if I comment out :69 the test still passes).  need to add `->expects($this->once())` to the `$endpoint` mock.

Reply:
remove the assertTrue() ?